### ISSUE TITLE
[Accessibility]A11y_VSCode_ColorTheme_ColorContrast:Color contrast ratio is less than 4.5:1 for the Blue colored content when focused for Editor screen

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#56796]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1)Navigate to Settings-->Select color theme -->select solarized light
2)Check color contrast ratio for the Editor content on the screen.

**Actual:**
Color contrast ratio is less than 4.5:1 for the black colored content when focused.
Similar issue is observed in following themes:
Solarized light
Solarized dark
Tomorrow night blue
Expected:
Color contrast ratio should be more than 4.5:1 for the content on a screen.
Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

](url)
**User Impact:**
The low vision/visually impaired users will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
](url)

**Attachment For Reference:**

![22855_a11y_vscode_colortheme_colorcontrast](https://user-images.githubusercontent.com/39204909/44326114-44acb480-a478-11e8-9288-82545d30019d.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/56797
